pkg/client/v1/client_test.go

```diff
@@ -125,6 +125,25 @@ func TestGetTableByID(t *testing.T) {
 	})
 }
 
+func TestBlockNumber(t *testing.T) {
```

Comment:
Created an e2e test for the block number in the client pkg. We have yet to find the best place for e2e tests. This package already has some, and it has some nice helpers that make it very quick to create an e2e test.